packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1583,14 +1583,20 @@ export async function renderToHTMLOrFlight(
                       src:
                         `${assetPrefix}/_next/` +
                         src +
-                        getAssetQueryString(false),
+                        // Always include the timestamp query in development
```

Comment:
Do you need to check `process.env.NODE_ENV === 'development'` here?

Reply:
It's included inside `getAssetQueryString`

Reply:
Nice!

Another option could be adding `Vary: *` if we want to try to preserve the url.